processor/apmprocessor/config.go

```diff
@@ -8,6 +8,8 @@ import (
 )
 
 type Config struct {
+	// If set to true, will set the `instrumentation.provider` attribute to `newrelic-opentelemetry`
```

Comment:
We should add a comment that the default is true

Reply:
👍 done